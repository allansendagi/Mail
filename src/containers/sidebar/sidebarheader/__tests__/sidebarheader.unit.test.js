import React from 'react';
import {shallow, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import SideBarHeader from '../sidebar-header.component';


configure({adapter: new Adapter()});
test('renders SideBarHeader', () => {
  const wrapper = shallow(
    <SideBarHeader/>
  );
  expect(wrapper).toMatchSnapshot();
});