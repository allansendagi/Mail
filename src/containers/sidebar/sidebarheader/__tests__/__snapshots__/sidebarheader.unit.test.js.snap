// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SideBarHeader 1`] = `ShallowWrapper {}`;
